OrbitCore/BlockChain.h

```diff
@@ -111,8 +111,30 @@ class BlockChain final {
  public:
   BlockChain() : size_(0) { root_ = current_ = new Block<T, BlockSize>(nullptr); }
 
+  BlockChain(const BlockChain& other) = delete;
+
+  BlockChain& operator=(const BlockChain& other) = delete;
+
+  BlockChain(BlockChain&& other) {
+    root_ = other.root_;
+    current_ = other.current_;
+    size_ = other.size_;
```

Comment:
set other.* to nullptr?

Reply:
also - add a test for blockchain moves.

Reply:
I reverted this change, as it is not needed here anymore.

Reply:
But for now, I made it explicit and deleted the move and copy operations